tasks/rsync-deploy.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: install rsync
+  apt: name=rsync state=present
+  sudo: true
+- name: ensure destination folder
+  file: path={{ platform_client_base_path }}/server/www state=directory
+- name: deploy by sync
+  synchronize:
+  args:
+    archive: no
+    recursive: yes
+    perms: yes
+    times: yes
+    src: "{{ platform_client_deploy_src }}/"
```

Comment:
Yes, it's a path accessible to rsync (which is run behind the scenes), which in most of the cases means local filesystem.